test/lib/ansible_test/_data/requirements/ansible-test.txt

```diff
@@ -1,4 +1,4 @@
 # The test-constraints sanity test verifies this file, but changes must be made manually to keep it in up-to-date.
 virtualenv == 16.7.12 ; python_version < '3'
-coverage == 6.5.0 ; python_version >= '3.7' and python_version <= '3.11'
+coverage == 7.2.7 ; python_version >= '3.7' and python_version <= '3.12'
```

Comment:
Now that 7.3.0 is out, we should use that. Since it drops Python 3.7 support we'll need a new entry here:

```suggestion
coverage == 7.3.0 ; python_version >= '3.8' and python_version <= '3.12'
coverage == 6.5.0 ; python_version >= '3.7' and python_version <= '3.7'
```